tests/proc/t17157.nim

```diff
@@ -1,5 +1,6 @@
 discard """
   errormsg: "'untyped' is only allowed in templates and macros or magic procs"
+  disabled: true
```

Comment:
@saem  why was this disabled? is that a regression?

Reply:
It was me who disabled it (after consultation with @Araq), not @saem.

And the reason is that it was more important to have this fix merged than this test green.